docs/book/src/topics/rosa/creating-a-cluster.md

```diff
@@ -5,6 +5,15 @@
 CAPA controller requires service account credentials to be able to provision ROSA clusters:
 1. Visit [https://console.redhat.com/iam/service-accounts](https://console.redhat.com/iam/service-accounts) and create a new service account.
 
+1. For every newly created service account, make sure to activate the account using the [ROSA command line tool](https://github.com/openshift/rosa). First, log in using your newly created service account
```

Comment:
better to be a sub item or note for the previous step.

Reply:
Fixed.